source/admin_manual/list_of_events.rst

```diff
@@ -854,579 +487,380 @@ These events apply only for connections using the ``connection API``.
-+---------------------+------------------------------------------------------+
 
-.. _event_fs_iter:
+.. dovecot_core:event:: fs_iter
 
-fs_file
--------
+   :field file_type: * ``mail``: Email file
+                     * ``index``: Index bundle
+                     * ``box``: Mailbox directory (for creating/deleting it,
+                       if used by the storage driver)
+                     * ``fts``: FTS file
+   :field reason: Reason for accessing the file.
+
+   Inherits from fs or any other specified event (e.g. mailbox).
+
+   .. todo:: obox events also have additional fields that need to be defined
+             here
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from :ref:`event_fs` or any other specified event                 |
-| (e.g. :ref:`event_mailbox`)                                                |
-+---------------------+------------------------------------------------------+
-
-If the file was created for obox, it has also fields:
-
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| file_type           | * mail: Email file                                   |
-|                     | * index: Index bundle                                |
-|                     | * box: Mailbox directory (for creating/deleting it,  |
-|                     |   if used by the storage driver)                     |
-|                     | * fts: FTS file                                      |
-+---------------------+------------------------------------------------------+
-| reason              | Reason for accessing the file                        |
-+---------------------+------------------------------------------------------+
 
 Storage
 =======
 
-.. _event_mail_storage_service_user:
+.. dovecot_event:field_group:: mail_storage_service_user
 
-Mail storage service user
--------------------------
+   :field session: Session ID for the storage session
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from environment (e.g. IMAP/LMTP client)                          |
-+---------------------+------------------------------------------------------+
-| session             | Session ID for the storage session                   |
-+---------------------+------------------------------------------------------+
+   .. todo:: mail_storage_service_user "Inherits from environment (e.g.
+             IMAP/LMTP client)"
```

Comment:
Like done elsewhere, how about just write text:

```
Inherits from any specified event (e.g. imap client).
```

Reply:
See above.  (I don't know what to do with these polymorphic events.)